### DISCUSSION TITLE
Taro发布到H5和小程序可以设置引导页嘛？

### DISCUSSION THREAD

**Discussion by 1164226245** [NervJS/taro#12935] [Feature Request]
在根目录app.jsx里面通过componentDidMount生命周期 Taro.redirectTo跳转到引导页能这样写嘛？

**Comment by TheKonka** [comment]
可以 引导页 作为首页？ 然后 自动跳转到真正的首页？

**Reply by 1164226245** [reply]
是的，如果用户是第一次访问，那就先进入引导页，然后再跳转真正的首页，若用户不是第一次登录进来就还是进入首页，就是不知道通过Taro.redirectTo跳转是否能成功
